cmake/add_enclave.cmake

```diff
@@ -183,8 +184,8 @@ function (add_enclave_sgx)
   # It only re-enables the additional LVI-mitigated build of the specified enclave.
   # It does not enable the additional ctest against the LVI-mitigated version of
   # the enclave.
-  if ((LVI_MITIGATION MATCHES ControlFlow)
-      AND (ENCLAVE_ADD_LVI_MITIGATION OR NOT LVI_MITIGATION_SKIP_TESTS))
+  if ((NOT LVI_MITIGATION MATCHES None) AND (ENCLAVE_ADD_LVI_MITIGATION
```

Comment:
Why did you change "Matches ControlFlow"

Reply:
Originally I was trying to support both ControlFlow and All option, so I changed them to check `NOT None`, to handle both ControlFlow and All.

Reply:
Can we switch it back so the reader knows we only support ControlFlow?